data_preparation/global_soil_moisture_GLDAS/README.md

```diff
@@ -0,0 +1,41 @@
+# Global Soil Moisture Data from GLDAS
+
+## Short description
+?
```

Comment:
Gridded and global soil moisture data developed by integrating satellite- and ground-based observational data products, using advanced land surface modeling and data assimilation techniques.